index.js

```diff
@@ -87,6 +87,7 @@ const convertGtfsToSql = async function* (files, opt = {}) {
 -- ${pkg.homepage}
 \\set ON_ERROR_STOP True
 CREATE EXTENSION IF NOT EXISTS postgis;
+CREATE SCHEMA IF NOT EXISTS ${opt.schema};
```

Comment:
Using a schema other than `public` allows us to cleanly check if data has been imported already, because a `CREATE SCHEMA` without `IF NOT EXISTS` would fail in this case.

What do you think about only adding `IF NOT EXISTS` if `opt.schema !== 'public'`?

Reply:
This is conditional now. 
`CREATE SCHEMA IF NOT EXIST` will run only if `opt.schema !== public`